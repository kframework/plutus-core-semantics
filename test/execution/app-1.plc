(program

(module Foo (import) (export () ())
  (declare r1 integer)
  (define r1
    (let (lam x (builtin addInteger x 5)) addFive [addFive 10]))
  (declare r2 integer)
  (define r2
    [(lam x (lam y (lam z (builtin addInteger x (builtin multiplyInteger y z))))) 2 3 5])
)

)
