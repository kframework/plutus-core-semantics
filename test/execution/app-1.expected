<T>
  <k> . </k>
  <lenv> .Map </lenv>
  <genv> ( Foo . r1 ) |-> 15 ( Foo . r2 ) |-> 17 </genv>
  <modules>
    <module>
      <curr> 0 </curr>
      <name> Foo </name>
      <data> .TyExpList </data>
      <decl> r2  ( r1  .NameList ) </decl>
      <import> .ModNmList </import>
      <exportTy> .TyExpList </exportTy>
      <exportTm> .NameList </exportTm>
    </module>
  </modules>
</T>
